clone-mdb-docs-repos.py

```diff
@@ -5,18 +5,36 @@
 tengen = "10gen"
 mongodb = "mongodb"
 
-try:
-    github_username = sys.argv[1]
-except:
-    print("################################################################################")
-    print("you did not include your github username.")
-    print("please run the script again and provide your github username as an argument.")
-    print("")
-    print("Example:")
-    print("")
-    print("python3", sys.argv[0], "jwilliams-mongo")
-    print("################################################################################")
-    exit()
+
+def try_to_get_sys_arg(argv_position, error_msg):
+    try:
+        return sys.argv[argv_position]
+    except:
+        print(error_msg)
+        exit()
+
+github_username = try_to_get_sys_arg(1, f"""
+################################################################################
+you did not include your github username.
+please run the script again and provide your github username as an argument.
+
+Example:
+python3 {sys.argv[0]} jwilliams-mongo /Users/jwilliams/projects/
+################################################################################
+""")
+
+
+path_to_repos = try_to_get_sys_arg(2, f"""
+################################################################################
+you did not include the desired path to repos.
+please run the script again and provide the desired path to your repos.
+
+Example:
+python3 {sys.argv[0]} jwilliams-mongo /Users/jwilliams/projects/
+################################################################################
+""")
+os.chdir(path_to_repos) # throws error if path doesn't exist
+
 
```

Comment:
consider adding a new list here that collects the repos that were successfully cloned. There's more output now so a summary when the command completes would be nice.